zeebe/clients/java/src/main/java/io/camunda/zeebe/client/impl/http/HttpClient.java

```diff
@@ -95,16 +94,37 @@ public <HttpT, RespT> void get(
       final Class<HttpT> responseType,
       final JsonResponseTransformer<HttpT, RespT> transformer,
       final HttpZeebeFuture<RespT> result) {
+    sendRequest(path, null, requestConfig, responseType, transformer, result);
+  }
+
+  public <HttpT, RespT> void post(
+      final String path,
+      final String body,
+      final RequestConfig requestConfig,
+      final HttpZeebeFuture<RespT> result) {
+    sendRequest(path, body, requestConfig, Void.class, r -> null, result);
+  }
+
+  private <HttpT, RespT> void sendRequest(
+      final String path,
+      final String body,
+      final RequestConfig requestConfig,
+      final Class<HttpT> responseType,
+      final JsonResponseTransformer<HttpT, RespT> transformer,
+      final HttpZeebeFuture<RespT> result) {
     final URI target = buildRequestURI(path);
-    final SimpleHttpRequest request = SimpleRequestBuilder.get(target).build();
+    final SimpleRequestBuilder requestBuilder = SimpleRequestBuilder.post(target);
```

Comment:
@koevskinikola, this is the issue, I would say. We're always sending a POST 🙈

Reply:
@ana-vinogradova-camunda I pushed a fix for the issue. Let me know if you can review the PR before we merge it.